docs/physics/energy_input/positronium.rst

```diff
@@ -0,0 +1,23 @@
+**************************
```

Comment:
line of *** is too long, should be ``***********`` on top and bottom (one * for each character in the title, otherwise sphinx complains).